draft-ietf-quic-qpack.md

```diff
@@ -523,6 +532,41 @@ blocks within a stream have been fully processed.
 ~~~~~~~~~~
 {:#fig-header-ack title="Header Acknowledgement"}
 
+An encoder MUST treat receipt of a Header Acknowledgment as also acknowledging
+any dynamic table entries that the header block referenced.  That is, this
+instruction is also processed as a Table Size Synchronize instruction with a
+value matching the Largest Reference of the corresponding header block.
+
+A decoder MUST track increases to the largest acknowledged dynamic table entry
+caused by acknowledging a header block so that it can correctly generate the
+Table State Synchronize instruction.
+
+
+### Stream Cancellation
+
+A stream that is reset might have multiple outstanding header blocks.  A decoder
+that receives a stream reset before the end of a stream generates a Stream
+Cancellation instruction on the decoder stream.  Similarly, a decoder that
+abandons reading of a stream needs to signal this using the Stream Cancellation
+instruction.  This signals to the encoder that all references to the dynamic
+table on that stream are no longer outstanding.
+
+An encoder cannot infer from this acknowledgement that any updates to the
```

Comment:
s/acknowledgement/instruction/ ?

Reply:
This builds on top of #1399 (You can view the changes here: https://github.com/quicwg/base-drafts/compare/implicit-ack-header-block...ack-reset-stream  I assumed - incorrectly - that you had already seen these changes, so pointed this at master rather than the other PR.